Components/AddTaskMenu.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { View, TextInput, TouchableOpacity, StyleSheet, Text, KeyboardAvoidingView } from 'react-native';
+
+export default class AddTaskMenu extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            text: '',
+            isDisabled: true,
```

Comment:
don't set bool to isDisabled
instead calculate it:
isDisabled = this.state.text.trim() != ''